<launch>
    <!-- //typefrom cmakelist node pkg from cmakelist -->
    <param name="log_folder"      type="string"  value="$(find tf_test)" />

     <param name="map_resolution"   type="double"  value="0.1" />
     <param name="map_width"             type="int" value="2000" />
     <param name="map_height"            type="int" value="2000" />
     <!-- <param name="originx"                       type="int" value="-250" />
     <param name="originy"                       type="int" value="-250" /> -->

     <node pkg="tf" type="static_transform_publisher" name="world_to_grid" args="0 0 0 0 0  0 /map_odom  /map 100"/>

    <!-- <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" /> -->
    <node name="cloud_map_node" pkg="tf_test" type="point_cloud_map" output="screen" />

     <!-- <node pkg="tf" type="static_transform_publisher" name="base_world_to_grid" args="-25 0 0 0 0  1 /world /grid  100"/> -->

    <!-- <node pkg="rviz" type="rviz" name="rviz" output="screen"
        args="-d $(find gpsmapping)/ros_wrapper/rviz/default.rviz" required="true">
    </node> -->



</launch>